src/main/frontend/src/components/aggregates/Route.svelte

```diff
@@ -15,8 +15,8 @@
 	export let id;
 
 	let requestMethodDisabled = false;
-	$: changedMethodOrMethods(aggregateMethod, methods);
-	const changedMethodOrMethods = (_, methods) => {
+	$: changedMethodOrMethods(methods);
+	const changedMethodOrMethods = (methods) => {
 		const method = methods.find(m => m.name === aggregateMethod);
```

Comment:
If you are against the argument for just reactive purposes, you can use the aggregateMethod argument right here (so instead of calling it "`_`" call it `aggregateMethod`